aiogram/utils/helper.py

```diff
@@ -135,69 +144,70 @@ def apply(cls, text, mode):
             return mode(text)
         return text
 
+class _BaseItem:
+    def __init__(self, value: Optional[str] = None):
+        self._value = cast(str, value)
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        if not name.isupper():
+            raise NameError("Name for helper item must be in uppercase!")
+        if not self._value:
+            # TODO: а если не имеет?
```

Comment:
What we should do if owner hasn't attribute "mode" and _value is None? 🤔

Reply:
@JrooTJunior

Reply:
Helper is very controversial feature, I do not know what to do with it

Reply:
Woops. It seems like bug in all previous versions of aiogram.

Originally Item was planned to use only as descriptor for Helper classes and in this case owner is always has the `mode` attribute.

I think need to add validation of parent
```python3
if not inspect.isclass(owner) or not issubclass(owner, Helper):
    raise RuntimeError("Item classes can be used only as Helper attributes")
```

Reply:
What about OrderedHelper? It isn't a subclass of Helper

Reply:
Okay, I fixed it with adding Helper as OrderedHelper parent